packages/rum-core/src/browser/domMutationObservable.ts

```diff
@@ -1,8 +1,64 @@
-import { monitor } from '@datadog/browser-core'
-import { LifeCycle, LifeCycleEventType } from '../domain/lifeCycle'
+import { monitor, Subscription } from '@datadog/browser-core'
 
-type MutationObserverConstructor = new (callback: MutationCallback) => MutationObserver
+export interface DOMMutation {
```

Comment:
What about renaming this interface to `DOMMutationObservable`, since it is the value returned by `createDOMMutationObservable` (as well as its usages `domMutation` -> `domMutationObservable`)? `DOMMutation` might be confused with the object emited by the `Observer` (`MutationRecord`).

Reply:
Yes that's a good point